mod-finance/schemas/fy_finance_data.json

```diff
@@ -0,0 +1,73 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Finance data for a fiscal year",
+  "type": "object",
+  "properties": {
+    "fiscalYearCode": {
+      "description": "The code of the fiscal year",
+      "type": "string"
+    },
+    "fundId": {
+      "description": "Fund UUID",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "fundCode": {
+      "description": "A unique code associated with the fund",
+      "$ref": "../../common/schemas/fund_code.json"
+    },
+    "fundName": {
+      "description": "The name of this fund",
+      "type": "string"
+    },
+    "fundStatus": {
+      "description": "The current status of this fund",
+      "type": "string",
+      "enum": [
+        "Active",
+        "Frozen",
+        "Inactive"
+      ]
+    },
+    "fundTags": {
+      "type": "object",
+      "description": "Arbitrary tags associated with this fund",
+      "$ref": "../../../raml-util/schemas/tags.schema"
+    },
+    "budgetId": {
+      "description": "UUID of this budget",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "budgetName": {
+      "description": "The name of the budget",
+      "type": "string"
+    },
+    "budgetStatus": {
+      "description": "The status of the budget",
+      "type": "string",
+      "enum": [
+        "Active",
+        "Frozen",
+        "Inactive",
+        "Planned",
+        "Closed"
+      ]
+    },
+    "budgetInitialAllocation": {
+      "default": 0,
+      "description": "The amount of the first allocation made to this budget",
+      "type": "number"
+    },
```

Comment:
we also need to include field budget.allocated per this screen
![image](https://github.com/user-attachments/assets/a117b3f4-a9ae-495d-b678-a0bee3b97838)

Reply:
yes, we need to provide both allocation per this csv file: initialAllocation and allocated
![image](https://github.com/user-attachments/assets/a1eeacbd-aab4-473a-8ff1-fd3d5211a05d)

Reply:
thanks to clarfiy, added allocated field as budgetCurrentAllocation